src/app/(home)/page.tsx

```
@@ -9,12 +9,29 @@ import HomePrompt from "./_components/HomePrompt";
-      <AppContainer className="h-100 w-100 flex align-center justify-center">
+      <AppContainer className="h-100 w-100 grid align-center justify-center">
         <section className="w-100 p-medium-60">
           <HomeHeading />
           <HomePrompt />
         </section>
       </AppContainer>
+
+      <footer className="p-medium-60  flex justify-center align-en g-medium-10">
+        <p className="fs-medium-10">
+          <span className="opacity-default-60">Built by</span>&nbsp;
+          <a target="_blank" href="https://github.com/nnsncl">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)